addons/l10n_es/data/account_tax_data.xml

```diff
@@ -4190,6 +4190,339 @@
                 'account_id': ref('l10n_es.account_common_4751'),
                 'tag_ids': [ref('mod_111_03')],
             }),
-        ]"/>
+    </record>
+    <record id="account_tax_template_s_irpfnrnue24" model="account.tax.template">
+        <field name="description">IRPF 24%</field>
+        <field name="type_tax_use">sale</field>
+        <field name="name">Retenciones a cuenta IRPF No residentes no-UE 24%</field>
+        <field name="active">False</field>
+        <field name="chart_template_id" ref="l10n_es.account_chart_template_common" />
+        <field name="amount" eval="-24" />
+        <field name="amount_type">percent</field>
+        <field name="tax_group_id" ref="tax_group_irnr_no_ue" />
+        <field
```

Comment:
It would not be bad to follow the same indentation as elsewhere: 
<field name="invoice_repartition_line_ids" eval="[(5, 0, 0),